src/style-auro-calendar.scss

```
@@ -0,0 +1,186 @@
+// // Copyright (c) 2022 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// // See LICENSE in the project root for license information.
+
+// // ---------------------------------------------------------------------
+
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/breakpoints';
+@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
+
+/* stylelint-disable color-function-notation, at-rule-empty-line-before */
+
+$mobile-header-height: 86px;
+$mobile-footer-height: 200px;
```

```suggestion
:root {
  --mobile-header-height: 86px;
  --mobile-footer-height: 200px;
}
```

Preferred that we move to CSS variables.

---

Declaring the variable this way at the style.scss level does not seem to actually create the variable for use within the file.

Also, using a variable name here seems questionable to me as we would need to rename all these to something we can be more sure is unique. Any number of components might end up declaring `--mobile-footer-height`.

---

I suggest issues against the design be opened to get these sections of the layout built using the design tokens. It's the kind of thing June is good at making sure she does on hers but this one was done outside our team.

We can do a UI iteration later to refine the layout a bit.